guides/source/active_record_querying.md

```diff
@@ -1459,7 +1459,7 @@ SELECT books.* FROM books
   INNER JOIN reviews ON reviews.book_id = books.id
 ```
 
-Or, in English: "return all books with their author that have at least one review". Note again that books with multiple reviews will show up multiple times.
+Or, in English: "return all books with an author that have at least one review". Note again that books with multiple reviews will show up multiple times.
```

Comment:
It's probably clearer if we group the author with the review:
```suggestion
Or, in English: "return all books that have at least one author and one review". Note again that books with multiple reviews will show up multiple times.
```

Reply:
I like your idea of putting them together.

I was wondering if we wanted to keep a distinction between books belonging to an author and having many reviews? It would read like this: 

Or, in English: "return all books that have an author and at least one review".  ?

(I don't know how to do a code suggested change so I've pushed up another version but feel free to ignore).

Reply:
> Or, in English: "return all books that have an author and at least one review". ?

That looks great!